runelite-client/src/main/java/net/runelite/client/game/ItemManager.java

```diff
@@ -328,10 +328,19 @@ public int getAlchValue(int itemID)
 
 	public int getBrokenValue(int itemId)
 	{
-		PvPValueBrokenItem b = PvPValueBrokenItem.of(itemId);
+		return getBrokenValue(itemId, false);
+	}
+
+	public int getBrokenValue(int itemId, boolean fullValue)
+	{
+		final PvPValueBrokenItem b = PvPValueBrokenItem.of(itemId);
```

Comment:
We can think of a better variable name than this ;)

Reply:
Also a better name for that class :$ `ItemReclaimCost`?